ci/verify-chefdk.sh

```diff
@@ -2,6 +2,9 @@
 
 set -evx
 
+# Cleanning up some cruft from previous tests
+sudo find /tmp -name 'chef-dk*' | sudo xargs rm -rf
```

Comment:
ok I just checked a tester and there are new directories from today that can only be touched with sudo so we'll keep the `sudo` for now